### DISCUSSION TITLE
Discord link

### DISCUSSION THREAD

**Discussion by tech-bash** [hyperledger/aries-vcx#810] [General]
The discord link provided doesn't work, can you help me with the new link for joining the community!

**Comment by mirgee** [comment]
Hi @tech-bash ! 👋 Thanks for notifying us about the issue, we will update the link. In the meanwhile, you can try [this](https://discord.gg/hyperledger) link - you will find us on the `aries-vcx` channel. Let me know if this link works for you!

**Comment by tech-bash** [comment]
Yep, this link works perfectly. Thanks :)
